packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/ProjectControllerIntegrationTests.java

```diff
@@ -256,4 +272,55 @@ public void givenAnInvalidProjectToDeleteWhenMutationIsPerformedThenErrorIsRetur
 
         assertThat(this.domainEventCollector.getDomainEvents()).hasSize(0);
     }
+
+    @Test
+    @DisplayName("Given a project, when the project is renamed, then a project event is emitted")
+    @Sql(scripts = {"/scripts/initialize.sql"}, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = {"/scripts/cleanup.sql"}, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))
+    public void givenProjectWhenTheProjectIsRenamedThenProjectEventIsEmitted() {
+        var projectEventInput = new ProjectEventInput(UUID.randomUUID(), TestIdentifiers.ECORE_SAMPLE_PROJECT);
+        var flux = this.graphQLRequestor.subscribe(GET_PROJECT_EVENT_SUBSCRIPTION, projectEventInput);
+
+        var input = new RenameProjectInput(UUID.randomUUID(), TestIdentifiers.ECORE_SAMPLE_PROJECT, "New Name");
+        Runnable renameProjectTask = () -> {
+            var result = this.graphQLRequestor.execute(RENAME_PROJECT_MUTATION, input);
+            String typename = JsonPath.read(result, "$.data.renameProject.__typename");
```

Comment:
My CheckStyle complains about 3 occurrences of `"$.data.deleteProject.__typename"`.